### ISSUE TITLE
Add closing price of tickers

### ISSUE THREAD

**Issue by spartan737** [frefrik/ark-invest-api#71]
Possible to add closing price of tickers for ETF Profile, Holdings and Trade?

I'm the creator of Stocksera [https://stocksera.pythonanywhere.com], have been using your API from the start. Thank you!

**Comment by frefrik** [comment]
Great to see the api being used and appreciated🙏
Awesome job with Stocksera 🙌 very cool project!

> Possible to add average purchasing price of tickers for ETF Profile, Holdings, historical & current trade?

Do you have any examples of desired output for those endpoints?

And, got any suggestion how we should calculate average purchasing price for stocks traded and held prior to the earliest date in my dataset (2020-09-09)?

`ETF Profile` doesn't include any trade-data in reference to purchasing price, but I guess you maybe meant `Stock Profile`?

**Comment by spartan737** [comment]
Yes stock profile, my bad. I guess the best we could do is to add the closing price of the ticker on the day it is bought.

Another thing I hope to have is to have a consolidated daily trade of all ARK funds (so that 1 request needs to be made instead of multiple requests for each ARK fund).

The last thing would be the combined weights of the tickers in all funds (same thing, so that 1 request needs to be made instead of multiple requests for each ARK fund).

**Comment by frefrik** [comment]
> Yes stock profile, my bad. I guess the best we could do is to add the closing price of the ticker on the day it is bought.

Added a new optional parameter to Stock Profile endpoint now to get the current share price in the same request: https://arkfunds.io/api/v2/stock/profile?symbol=TSLA&price=true

> Another thing I hope to have is to have a consolidated daily trade of all ARK funds (so that 1 request needs to be made instead of multiple requests for each ARK fund).

Implemented 👌🏻 
You can now pass multiple fund symbols in comma separated string for endpoints `ETF Holdings`, `ETF Trades`, `ETF News`: https://arkfunds.io/api/v2/etf/trades?symbol=ARKK,ARKX,ARKQ

> The last thing would be the combined weights of the tickers in all funds (same thing, so that 1 request needs to be made instead of multiple requests for each ARK fund).

I'll consider to add this functionality if I get time. But for now, with the new changes, you can query all current holdings in 1 request and do the calculation client-side.
https://arkfunds.io/api/v2/etf/holdings?symbol=ARKK,ARKQ,ARKW,ARKG,ARKF,ARKX,PRNT,IZRL
